src/pages/Restorans/Menu/Menu.jsx

```diff
@@ -0,0 +1,129 @@
+import { React } from 'react';
+import HeaderRestoran from '../../../components/Header-Restoran/Header-Restoran';
+import Nav from '../../../components/Nav/Nav';
+import logoMeni from '../../../images/logo-for-meni.png';
+import FooterForRestoran from '../../../components/Footer/FooterForRestoran';
+import itemsMeni from '../../../components/data/menuitems';
+import ButtonMenu from '../../../components/ButtonMenu';
+import { useDispatch, useSelector } from 'react-redux';
+import { addFood } from '../../../store/actions/food';
+import removefood from '../../../images/removefood.svg';
+
+const Menu = () => {
+  const dispatch = useDispatch();
+  const { sum, foods } = useSelector(({ food }) => food);
+
+  return (
+    <div className='main-meni'>
+      <div className='navigation-menu-for-restoran'>
+        <HeaderRestoran />
+      </div>
+      {/* <div className='container'>
```

Comment:
This is `<ul>` that we discussed to be removed, so just remove comment

Reply:
Deleted